examples/workers/workers.bal

```diff
@@ -9,23 +9,24 @@ public function main() {
 
     // This block belongs to the `w1` worker.
     worker w1 {
-        http:Client httpClient = new ("https://api.mathjs.org");
-        var response = httpClient->get("/v4/?expr=2*3");
+        http:Client httpClient = new("https://api.mathjs.org");
+        var response = httpClient -> get("/v4/?expr=2*3");
```

Comment:
Formatting shouldn't change these AFAIK. 

@SupunS, @IrushiL, thoughts please?

Reply:
The VSCode plugin changed the formatting

Reply:
This new formatting follows the ballerina formatting guidelines.. So we can go ahead with those unless we find any obvious issues.

However, we don't need to add the additional newline at the end. That was due to a bug in the shared pack: https://github.com/ballerina-platform/ballerina-lang/issues/26002.

Reply:
Have the guidelines changed? The convention was to not use spaces around `->` for remote method calls right? Given that it is a method call, just that we use `->` instead of `.`?

For `new ()` also, given that it is like `new Foo()`, at one point we decided on having a space between `new` and `()`, I can't seem to recall any documentation though.

Reply:
There is this doc https://docs.google.com/document/d/1dQ1Z_fNHgrdGqp5Nkf2aBar97kjlo2kjIf71O34u6j0/edit as the guide. But seems the remote action is undefined.

Reply:
If the remote call is something that we have already discussed and decided, +1 to go ahead with that.
Re: `new`, yes, seems there should be a space in between, as per the doc.

Reply:
Noted, thanks! Should we also create issues for the formatter for these changes?

Re `->`, yeah, I believe the convention was to
- have spaces around `->` for worker send
- not have spaces around `->` for remote method calls

Looping in @anupama-pathirage too.

Reply:
shall I create an issue for the remote actions and revert the changes? And for the new as well?

Reply:
@MaryamZi @SupunS As per the discussion, I made the following changes in PR [1].
* `new()` changed to `new ()` in `ImplicitNewExpressionNode`
* spaces around `->` removed in `RemoteMethodCallActionNode`
* spaces after `{` and before `}` are removed in inline braces

[1] https://github.com/ballerina-platform/ballerina-lang/pull/26529